lib/common/timers.ts

```diff
@@ -60,6 +67,9 @@ export function patchTimer(window: any, setName: string, cancelName: string, nam
   clearNative =
       patchMethod(window, cancelName, (delegate: Function) => function(self: any, args: any[]) {
         var task: Task = args[0];
+        if (task && typeof task === 'number') {
+          task = tasksByHandleId[<number>task];
+        }
```

Comment:
Done.

Can’t drop the `<number>` type casting in the `tasksByHandleId[<number>task]` index expression, though, the TypeScript compiler gives error.